CommunicationLayer/CommunicationContainer.cpp

```diff
@@ -2,12 +2,9 @@
 #include <QUdpSocket>
 
 #include "../DataLayer/DataContainer.h"
-#include "CommunicationContainer.h"
-#include "CommDeviceControl/ConnectionController.h"
-#include "CommDeviceControl/CommDeviceManager.h"
 #include "CommDeviceControl/RadioConnectionService.h"
-#include "CommDeviceControl/UdpConnectionService.h"
```

Comment:
I think we still need this, we will be connecting to the server through 3G with UDP. But it is true the logic in there doesn't make sense, I think we should keep the file but turn all the functions into stubs.